website/src/pages/privacy/cookie-policy.tsx

```diff
@@ -0,0 +1,46 @@
+import ArrowRightIcon from 'mdi-react/ArrowRightIcon'
+import * as React from 'react'
+import { Helmet } from 'react-helmet'
+import Layout from '../../components/Layout'
+export default class CookiesPage extends React.Component<any, any> {
+    constructor(props: any) {
+        super(props)
+        this.state = {
+            cookies: [],
+        }
+    }
+
+    public componentDidMount(): void {
```

Comment:
Can we address this hack with CSS?

Reply:
@aileenrose Why is this needed? Can it be removed?

Reply:
I'm not sure to do that.

Reply:
removed.

Reply:
wasn't clear about the hack with CSS you were referring to in the previous comment.